cluster-up/cluster/okd-4.1.0-nodes3/README.md

```diff
@@ -0,0 +1,45 @@
+# OKD 4.1.0-rc.0 in ephemeral containers
```

Comment:
4.1.0-nodes3

Reply:
ah, i need to remove this provider. It was an experiment I was working with. It isn't ready for consumption yet though.